spec.bs

```diff
@@ -705,6 +706,19 @@ script should likely be pulled out:
 [2]:
 https://html.spec.whatwg.org/multipage/document-lifecycle.html#shared-document-creation-infrastructure
 
+<div algorithm>
+
+To <dfn>generate reporting URLs</dfn> given an {{AuctionAdConfig}} |config| and a [=generated bid=]
+|bid|:
+1. TODO: Fetch buyer reporting script...though first we need the IG.
+1. TODO: Calculate all params to reportWin() using |bid|.
+1. TODO: Evaluate reportWin() with params and scope that includes sendReportTo().

```

Comment:
Can you explain what "scope that include sendReportTo()"? Is that another function we're supposed to evaluate or is this something else?

Reply:
sendReportTo() is provided by the platform, I think the WebIDL is over in #505